rakefile

```diff
@@ -577,8 +629,24 @@ namespace :pe_xl do
   file "#{BUILD_DIR}/params.json" do
     puts "Provisioning nodes for pe_xl"
     raise KeyError, 'key not found: "BEAKER_PE_VER"' unless ENV["BEAKER_PE_VER"].is_a? String
+
+    m = ENV["BEAKER_PE_VER"].match(/(^\d{4}\.\d+)\.(\d+)(.*$)/i)
```

Comment:
A more descriptive variable name than the single letter `m` would improve readability.

Reply:
Updated.